components/footer.js

```diff
@@ -0,0 +1,109 @@
+export default function Footer() {
+  let year = new Date().getFullYear();
+
+  return (
+    <footer className="footer bg-white pt-1 text-gray">
+      <div className="container mx-auto px-6 border-t-2">
+        <div className="text-center py-6">
+          <div className="inline-flex text-black">
+            <a href="" className="px-4">
```

Comment:
social links should target `_blank` to open in a new window. with `rel="noopener noreferrer"`